index.html

```diff
@@ -20,12 +20,123 @@
+            <h3 class="project_name">
+                Multi-Post
+                Stories
+            </h3>
+
+            <ul class="first_list">
+                <li>CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+                
+            </ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+        </ul>
+       <button class="btn1" type="submit">See Project</button>
+    </div>
+    <div class="work_card">
+       
+        <img src="./images/firstimage.png" alt="card"/>
+        <h3 class="project_name">
+            Multi-Post
+            Stories
+        </h3>
+
+        <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+            
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="tech_list">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+    </ul>
+    <button class="btn1" type="submit">See Project</button>
+</div>
+
+<div class="work_card">
+       
+    <img src="./images/thirdimage.png" alt="card"/>
+    <h3 class="project_name">
+        Multi-Post
+        Stories
+    </h3>
+
+    <ul class="first_list">
+        <li>CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+        
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+<ul class="tech_list">
+    <li>html</li>
+    <li>css</li>
+    <li>javaScript</li>
+</ul>
+<button class="btn1" type="submit">See Project</button>
+</div>
+<div class="work_card">
+       
+    <img src="./images/fourthimage.png" alt="card"/>
+    <h3 class="project_name">
+        Multi-Post
+        Stories
+    </h3>
+
+    <ul class="first_list">
+        <li>CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+        
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+<ul class="tech_list">
+    <li>html</li>
+    <li>css</li>
+    <li>javaScript</li>
+</ul>
+<button class="btn1" type="submit">See Project</button>
+</div>
+
+  
+
+<section>
     <div class="home_container">
-    <h1>I’m Shahnawaz Ali
-        Glad to see you!</h1>
+    <h1>About<br>Myself</h1>
 
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         <span>LET’S CONNECT</span>
```

Comment:
- [ ] Kindly make sure that your design match the Figma template design as much as possible (e.g. font, colors, and margins)
Your design
![Capture](https://user-images.githubusercontent.com/85095702/159096057-9591887d-6aa1-48b8-bb76-2716ae73c570.PNG)

Figma template
![figma](https://user-images.githubusercontent.com/85095702/159096076-32a0156a-d3a4-4f0d-b69b-e563c244e43a.PNG)